addons/utm/migrations/13.0.1.0/openupgrade_analysis.txt

```diff
@@ -14,7 +10,11 @@ utm          / mail.mass_mailing.tag    / __last_update (datetime)      : previo
 utm          / mail.mass_mailing.tag    / color (integer)               : previously in module mass_mailing
 utm          / mail.mass_mailing.tag    / display_name (char)           : previously in module mass_mailing
 utm          / mail.mass_mailing.tag    / name (char)                   : previously in module mass_mailing
+utm          / utm.campaign             / color (integer)               : NEW
```

Comment:
No more `previously in`?

Reply:
Yeah, it seems now it failed to recognize they were from mass_mailing module. I think it's due to the apriori renames.